cli/providers/aws/logs/logs.go

```diff
@@ -0,0 +1,31 @@
+package logs
+
+import (
+	"fmt"
+	logger "log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+	"github.com/pchico83/i2kit/cli/schemas/service"
+)
+
+//Logs prints the logs asociatted to a given instance
+func Logs(s *service.Service, startTime *int64, config *aws.Config, log *logger.Logger) error {
```

Comment:
could we call this something else to avoid confusion with the stock `log` package? (same for the package). Maybe this should be called WriteLogs() or ProcessLogs() instead?